### ISSUE TITLE
Add telemetry for the 'Default Browser' Local Notification 

### ISSUE THREAD

**Issue by Amejia481** [mozilla-mobile/fenix#19847]
https://jira.mozilla.com/browse/FNX-21798 related https://github.com/mozilla-mobile/fenix/issues/19846

**Comment by mcarare** [comment]
Waiting on https://github.com/mozilla-mobile/fenix/issues/19846.

**Comment by amedyne** [comment]
@rocketsroger 's PR for #19846 is almost ready.

**Comment by lobontiumira** [comment]
Verified as fixed on the 91.0.0-beta.2 build with Google Pixel (Android 10):
Glean ping to URL: /submit/org-mozilla-firefox-beta/events/1/968466e8-088a-4659-97b7-72473e6855a0
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/mira-210720002)

Verified on 92.0a1 Nightly from 7/19 also, with Google Pixel (Android 10):
Glean ping to URL: /submit/org-mozilla-fenix/events/1/d8fd3c8a-b6d8-43c1-b38a-978eda220bcb

```
 {
          "category": "events",
          "name": "default_browser_notif_tapped",
          "timestamp": 0
        }
```
